examples/legacycdkeysdumper.js

```diff
@@ -0,0 +1,61 @@
+const SteamUser = require("steam-user");
+const fs = require("fs");
+
+const client = new SteamUser({
+    enablePicsCache: true,
+});
+
+client.on("loggedOn", () => {
+    console.log("Logged on");
+});
+
+client.on("ownershipCached", async () => {
+    let apps = client.getOwnedApps({
+        excludeExpiring: false,
+        excludeFree: false,
+        excludeShared: false
+    }).filter(appid => {
+        // these keys should indicate presence of a legacy cd key
+        return [
+            "hadthirdpartycdkey",
+            "legacykeydisklocation",
+            "legacykeyfromapp",
+            "legacykeylinkedexternally",
+            "legacykeyproofofpurchaseticket",
+            "legacykeyregistrationmethod",
+            "legacykeyregistrylocation",
+            "showcdkeyinmenu",
+            "showcdkeyonlaunch",
+            "supportscdkeycopytoclipboard",
+            "thirdpartycdkey",
+        ].some(key =>
+            Object.keys(client.picsCache?.apps[appid]?.appinfo?.extended || {}).map(k => k.toLowerCase()).includes(key)
```

Comment:
Optional chaining is not available prior to node 13.2.0, and the minimum node version in package.json is 8.0.0. We shouldn't use features not available in the minimum supported node version.